jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/ThreadMXBeanImpl.java

```diff
@@ -756,4 +779,44 @@ public ObjectName getObjectName() {
 		return objectName;
 	}
 
+	private static MethodHandle getThreadInfoConstructorHandle() {
+		return AccessController.doPrivileged(new PrivilegedAction<MethodHandle>() {
+			@Override
+			public MethodHandle run() {
+				MethodHandle myHandle = null;
+				try {
+					Constructor<ThreadInfo> threadInfoConstructor = ThreadInfo.class
+							.getDeclaredConstructor(ThreadInfoBase.class);
+					threadInfoConstructor.setAccessible(true);
+					myHandle = MethodHandles.lookup().unreflectConstructor(threadInfoConstructor);
+				} catch (NoSuchMethodException | SecurityException | IllegalAccessException e) {
+					/*[MSG "K0614", "Unexpected exception accessing ThreadInfo constructor."]*/
```

Comment:
`K0614` is already defined in com/ibm/java/lang/management/internal/ManagementUtils.java
A new message number is needed and must only be defined once in this file.

Reply:
Created a separate message.

Reply:
Somebody added that message to the Java without adding it to `ExternalMessages-MasterIndex.properties`.  I will fix that.